docs/src/define_environment.md

```diff
@@ -2,6 +2,30 @@
 
 An [`Environment`](@ref) is a convienient object for applications like reinforcement learning and trajectory optimization. 
 
+For this example we need the following setup:
+```julia
+using Pkg
+Pkg.activate(joinpath(@__DIR__, ".."))
```

Comment:
These should not be necessary, and if they are we need to fix those.

Reply:
You are correct it was not necessary. I will remove that.